control-plane/agents/src/bin/core/node/wrapper.rs

```diff
@@ -82,7 +82,7 @@ pub(crate) type ReplicaSnapshotInfo = SnapshotInfo<ReplicaId>;
 #[derive(Debug, Clone)]
 pub(crate) struct NodeWrapper {
     /// The inner Node state.
-    node_state: NodeState,
+    pub(crate) node_state: NodeState,
```

Comment:
There's a getter function for this already

```suggestion
    node_state: NodeState,
```